ProffieOS.ino

```diff
@@ -18,8 +18,14 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-// You can have multiple configuration files, and specify which one
-// to use here.
+/*-------------------------------------------------------------------
+-  You can have multiple configuration files, and specify which one -
+-  to use here by removing the comment slashes at the beginning.    -
+-  **NOTE** Only ONE line should be left uncommented at a time!     -
+-  Add the slashes to any that you are not uploading.               -
+*/-------------------------------------------------------------------
```

Comment:
Does this compile?

Reply:
Embarrassingly, no.
It does now though.